sources/academy/webscraping/scraping_basics_javascript/data_extraction/project_setup.md

```diff
@@ -39,7 +39,7 @@ Node.js and npm support two types of projects, let's call them legacy and modern
 
 > More recent versions of npm might already have `"type": "commonjs",` pre-defined; if so, simply replace `commonjs` with `module`.
 
-> If you want to learn more about JSON and its syntax, we recommend [this tutorial on MDN](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Objects/JSON).
+If you want to learn more about JSON and its syntax, we recommend [this tutorial on MDN](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Objects/JSON).
```

Comment:
I think this was supposed to be admonition of some kind, but let it be, I'm about to go through the course and fix everything anyway.